packages/base-controller/CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.1.0]
+### Added
+- `update` function updated to expose patches, added `applyPatches` function to BaseControllerV2 ([#980](https://github.com/MetaMask/controllers/pull/980))
```

Comment:
Changelog entries are allowed to be split up -- in other words, there doesn't have to be a 1:1 correlation between entries and PR. One of these is a change, but the other is an addition. What do you think about splitting this entry up into both the Changed and Added sections?

Also (this is a nit) it's a convention to use the imperative for these entries (so, "add" vs. "added") as well as the active tense ("update `update` to..." vs. "`update` updated to...").

Reply:
Sounds good to me, I just split into Changed and Added, also updated the language.